src/cpp/jank/runtime/obj/transient_vector.cpp

```diff
@@ -0,0 +1,118 @@
+#include "jank/runtime/detail/object_util.hpp"
+#include "jank/runtime/erasure.hpp"
+#include "jank/type.hpp"
```

Comment:
Please use angle brackets <> for all jank includes. However, I don't think you actually need these includes, since they're in the pre-compiled header (PCH).